dotnet.cmd

```diff
@@ -20,10 +20,18 @@ if NOT [%ERRORLEVEL%]==[0] (
   exit /b %ERRORLEVEL%
 )
 
-pushd %~dp0
-echo Running: dotnet %*
-call "%~dp0\.dotnet\dotnet.exe" %*
-popd
+if "%DOTNET_INSTALL_DIR%"=="" (
```

Comment:
A nit - to avoid duplication of the code, you could just set a local variable to either ~dp0 or %DOTNET_INSTALL_DIR% and then have the rest of the code common.